apple-music.ts

```diff
@@ -1,4 +1,5 @@
-import { run } from "https://raw.githubusercontent.com/NextFire/jxa/v0.0.4/run/mod.ts";
+// import { run } from "https://raw.githubusercontent.com/NextFire/jxa/v0.0.4/run/mod.ts";
+import { run } from "./jxa.ts";
```

Comment:
I don't want to vendor dependencies. I think it should be fine to keep using the deprecated API until Deno 2. We can switch to another jxa dep

Reply:
Yep, done